frontend/src/views/SecretMainPage/components/CreateSecretForm/CreateSecretForm.tsx

```diff
@@ -73,6 +75,20 @@ export const CreateSecretForm = ({
     }
   };
 
+  const handlePaste = (e: ClipboardEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    const pastedContent = e.clipboardData.getData('text');
+    const splitIndex = pastedContent.indexOf("=");
+
+    if (splitIndex === -1) return
```

Comment:
This will make users unable to paste _just_ the key without a value.

To fix this we could do:

```js
if (splitIndex === -1) {
   setValue("key", pastedContent);
   return;
}

Reply:
nice catch